fastpass/src/main/scala/scala/meta/internal/fastpass/pantsbuild/commands/CreateCommand.scala

```diff
@@ -45,11 +47,19 @@ object CreateCommand extends Command[CreateOptions]("create") {
     val name = create.actualName
     Project.fromName(name, create.common) match {
       case Some(value) =>
-        app.error(
+        app.warn(
           s"can't create project named '${name}' because it already exists." +
-            s"\n\tTo refresh the project run: fastpass refresh ${name}"
+            s"\n\tTo refresh the project run: 'fastpass refresh ${name}'" +
+            s"\n\tTo open the project run: 'fastpass open --intellij ${name}'" +
+            s"\n\t                     or: 'fastpass open --vscode ${name}'"
         )
-        1
+        if (create.open.intellij) {
```

Comment:
```suggestion
        if (!create.open.isEmpty) {
          create.open.run(...)
        } else {
          app.error("can't create ... <old message>")
        }
```

Reply:
A bit confusing that I used "suggested changes", it was only meant as a rough draft